### PR TITLE
Task 6

### PR DESCRIPTION
**Pull request by pmvald**
-modified the card.

### CODE REVIEW

File: -sass/project-cmps/_prop-cards.scss [hunk 1]
```diff
@@ -75,13 +75,43 @@
     }
   }*/
 
+  &__mls{
+    position: absolute;
+    top: 5%;
+    left: 2.5%;
```
Comment (pmvald) [inline -sass/project-cmps/_prop-cards.scss:81]: Not sure if % unit is good. Maybe just use px unit.

File: -sass/project-cmps/_prop-cards.scss [hunk 2]
```diff
@@ -75,13 +75,43 @@
...
     color: black;
     padding: 0.25rem 0.5rem 0.5rem 0.5rem;
-
+    
     * {
-      @include dot-text-overflow();
+      // @include dot-text-overflow();
+    }
+     
+    &--addressCtn {
```
Comment (pmvald) [inline -sass/project-cmps/_prop-cards.scss:106]: 1. for names we use kebab case, like address-ctn
2. Why do we need this modifier?

File: -sass/project-cmps/_prop-cards.scss [hunk 3]
```diff
@@ -116,9 +147,43 @@
...
+
   &:focus {
     //outline: none;
     border: 1px solid red;
-  }
-
-}
+    box-shadow: 0 0 0 rgb(226, 26, 36);
+    animation: pulse 2s infinite;
+    @-webkit-keyframes pulse {
```
Comment (pmvald) [inline -sass/project-cmps/_prop-cards.scss:163]: Good! and could you explain why using both -webkit-keyframes and keyframes?

File: tasks/task-006/card.html [hunk 4]
```diff
@@ -1,64 +1,71 @@
...
-</head>
-<body>
-
-<div class="container">
-    <div style="width:250px; height:270px; padding:15px; border:2px solid black">
-
+    <link rel="stylesheet" href="../../css/bundle.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css"
```
Comment (pmvald) [inline tasks/task-006/card.html:9]: at the moment we are looking for some icon source that can be used offline. Can you use bootstrap icons offline?

File: tasks/task-006/card.html [hunk 5]
```diff
@@ -1,64 +1,71 @@
...
+            </div>
+            <img src="img.jpg" alt="Image" />
+          </div>
+          <div class="ss-prop-card__content">
+            <div class="ss-prop-card__content--addressCtn">
+              <!-- <span class="ss-prop-card__date" ~roles="login-pl/on-login-show">
+                [1021 day(s) ago] -
+              </span> -->
+              <span class="ss-prop-card__address me-q">
+                80 10199 RIVER DRIVE Lorem ipsum dolor sit, amet consectetur
```
Comment (pmvald) [inline tasks/task-006/card.html:39]: No need for "Lorem ipsum" text here. We already have the address and this is enough.

**Review by pmvald — COMMENTED** [review]
Good! I made code review

**Comment by pmvald** [comment]
@YusukeYoshihiro Hi, I have made code review but guess you didn't get the notification. Please make the changes.

File: -sass/project-cmps/_prop-cards.scss (hunk 1 above)
Comment (YusukeYoshihiro) [reply -sass/project-cmps/_prop-cards.scss:81]: I changed to `px` from `% persent`.

File: -sass/project-cmps/_prop-cards.scss (hunk 2 above)
Comment (YusukeYoshihiro) [reply -sass/project-cmps/_prop-cards.scss:106]: I just wanted to try to arrange the position of the map link as below.

Screen Shot 2021-06-07 at 2 54 57 PM

File: -sass/project-cmps/_prop-cards.scss (hunk 3 above)
Comment (YusukeYoshihiro) [reply -sass/project-cmps/_prop-cards.scss:163]: The reason is `@-webkit-keyframes` possible to complement and activate CSS animation for any other browsers especially `Safari`, most of the users ,who have iPhone devices and use `Safari`, sometimes are not able to see CSS animation without `webkit`. That is why we need to use both to complement CSS animation for all browsers to activate.

File: tasks/task-006/card.html (hunk 4 above)
Comment (YusukeYoshihiro) [reply tasks/task-006/card.html:9]: Okay, I will push my current change files and after I will pull the folders from the main branch.

File: tasks/task-006/card.html (hunk 5 above)
Comment (YusukeYoshihiro) [reply tasks/task-006/card.html:39]: Okay, I removed it.

**Comment by YusukeYoshihiro** [comment]
@pmvald Hi, Peter! I have updated `Task 6` so far and pushed it. Please confirm it.

File: tasks/task-006/card.html [hunk 6]
```diff
@@ -1,64 +1,67 @@
...
+            <div class="ss-prop-card__mls">
+              <p>MLS® #: R2586154</p>
+            </div>
+            <div class="ss-prop-card__brokerage"> 
+                <p >Nu Stream Realty Inc.</p>
+            </div>
+            <img src="img.jpg" alt="Image" />
+          </div>
+          <div class="ss-prop-card__content">
+            <div class="ss-prop-card__content--address-ctn">
```
Comment (pmvald) [inline tasks/task-006/card.html:34]: just use `ss-prop-card__address` ...

File: -sass/project-cmps/_prop-cards.scss [hunk 7]
```diff
@@ -75,13 +75,43 @@
...
     color: black;
     padding: 0.25rem 0.5rem 0.5rem 0.5rem;
-
+    
     * {
-      @include dot-text-overflow();
+      // @include dot-text-overflow();
+    }
+     
+    &--address-ctn {
```
Comment (pmvald) [inline -sass/project-cmps/_prop-cards.scss:106]: just use the bootstrap d-flex in the html ...

**Review by pmvald — COMMENTED** [review]
Good, I'm gonna improve and merge

**Comment by pmvald** [comment]
Good

**Pull request merged**